src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -64,6 +65,20 @@ public static int Main(string[] args)
                 return ConsoleRunner.INVALID_ARG;
             }
 
+            if (!string.IsNullOrEmpty(Options.ConsoleEncoding))
+            {
+                try
+                {
+                    Console.OutputEncoding = Encoding.GetEncoding(Options.ConsoleEncoding);
```

Comment:
Here is an example showing how unicode characters look in a Windows CMD console with the default font. The first example shows how unicode characters would be displayed with NUnit currently, and the second shows how they are displayed with UTF-8.

Unicode Text: `•ÑÜńĭŧ·`
CMD Font: "Raster Fonts" (bitmap)

NUnit > Default Encoding (CP437)
<img src="https://cloud.githubusercontent.com/assets/17755767/18958182/ca66c0c0-862f-11e6-83d7-0f5aa519ee1e.PNG" width="460"/>

NUnit > UTF-8
<img src="https://cloud.githubusercontent.com/assets/17755767/18958242/00a3eb54-8630-11e6-94d9-705d132345ef.PNG" width="460"/>